src/generate.rs

```diff
@@ -12,6 +12,7 @@ pub struct GenParams {
 #[derive(PartialEq, Eq)]
 pub enum GenMode {
     Get,
+    GetByCopy,
```

Comment:
In fitting with `GetMut` could we use `GetCopy`?

Reply:
It's OK